resources/clojupyter/assets/version.edn

```diff
@@ -1 +1 @@
-{:version "0.3.2"}
+{:version "0.3.3-SNAPSHOT"}
```

Comment:
Should this match `"0.4.0-SNAPSHOT"` in project.clj?

Reply:
Yes, we can match it.

As I see it, `version.edn` is read by `version.clj`, which itself was used by the cmdline interface. All those ended up being removed in the fork.